modules/optimization/src/base/OptimizationApp.C

```diff
@@ -46,6 +44,14 @@ OptimizationApp::registerAll(Factory & f, ActionFactory & af, Syntax & syntax)
       "AddOptimizationReporterAction", "OptimizationReporter", "add_optimization_reporter");
   registerMooseObjectTask("add_optimization_reporter", OptimizationReporter, false);
   addTaskDependency("add_optimization_reporter", "add_reporter");
+
+  // General Optimization action
+  registerTask("auto_create_mesh", false);
+  registerTask("auto_create_problem", false);
+  registerSyntaxTask("OptimizationAction", "Optimization", "auto_create_mesh");
+  registerSyntaxTask("OptimizationAction", "Optimization", "auto_create_problem");
```

Comment:
This syntax path is problematic. The new standard for MOOSE is to nest all actions defined in a module underneath the module name. This means that this root level action syntax is triggered for every other optimization action.

Reply:
how do I fix this?